code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -274,6 +319,16 @@
 	icon_state = "46x30mmtI"
 	ammo_type = /obj/item/ammo_casing/c46x30mm/inc
 
+/obj/item/ammo_box/magazine/wt550m9/empty
+	name = "wt550 magazine (4.6x30mm)"
+	icon_state = "46x30mmt"
+	ammo_type = /obj/item/ammo_casing/c46x30mm
+
+/obj/item/ammo_box/magazine/wt550m9/empty/New()
```

Comment:
Please use `Initialize()`

Reply:
I'm dumb ahahaha, that should have been obvious